### PR TITLE
Eventy stuff and pet protections!

### PR DESCRIPTION
**Pull request by Very-Soft**
Adds a BS dog and a stage prop for use in events.

Oh also makes most of the station pets not edible, and updates Apple to be a proper pet mob.

### CODE REVIEW

File: code/modules/mob/living/simple_mob/subtypes/animal/passive/mouse_vr.dm [hunk 1]
```diff
@@ -50,3 +50,13 @@
 			return 0
 	else
 		..()
+/mob/living/simple_mob/animal/passive/mouse/white/apple
+	name = "Apple"
+	desc = "Dainty, well groomed and cared for, her eyes glitter with untold knowledge..."
```
Comment (KasparoVy) [inline code/modules/mob/living/simple_mob/subtypes/animal/passive/mouse_vr.dm:55]: Forbidden!

**Comment by KasparoVy** [comment]
Hey @Very-Soft , woof icon state's west-facing dir seems to have 1 frame where it faces east judging by IconDiffBot

File: code/modules/mob/living/simple_mob/subtypes/animal/passive/mouse_vr.dm (hunk 1 above)
Comment (Very-Soft) [reply code/modules/mob/living/simple_mob/subtypes/animal/passive/mouse_vr.dm:55]: :3c

**Comment by Very-Soft** [comment]
> Hey @Very-Soft , woof icon state's west-facing dir seems to have 1 frame where it faces east judging by IconDiffBot

Good catch, thanks!

**Comment by KasparoVy** [comment]
No worries, you make nice sprites so ofc I had a look!

**Comment by Very-Soft** [comment]
Okay I'm done fiddling with this for now

**Review by Novacat — CHANGES_REQUESTED** [review]
I would like stage.dm and donteatpets.dm to have _vr added to the end of them, turning them into stage_vr.dm and donteatpets_vr.dm. This makes it easier to know which files are polaris and which are virgo.

**Review by Novacat — APPROVED** [review]

**Pull request merged**